src/chrome/content/rules/CIMB-Indonesia.xml

```diff
@@ -0,0 +1,48 @@
+<!--
+
+	Hosts may take a long time to respond.
+
+	Other CIMB Group rulesets:
+		- CIMB-Malaysia.xml
+		- CIMB-Singapore.xml
+		- CIMB-Thailand.xml
+		- CIMB-Cambodia.xml
+
+	Non-functional domain:
+		- (www.)cimbniagasyariah.com		(Mismatch hostname, CN: www.cimbniaga.com)
+		- (www.)bizchannel.cimbniaga.co.id	(site unreachable)
+
+-->
+
+<ruleset name="CIMB Indonesia">
+
+	<target host=    "cimbniaga.com" />
+	<target host="www.cimbniaga.com" />
+	<target host=    "cimbclicks.co.id" />
+	<target host="www.cimbclicks.co.id" />
+	<target host=    "itradecimb.co.id" />
+	<target host="www.itradecimb.co.id" />
+
+	<securecookie host="^www\.cimbclicks\.co\.id$" name=".*" />
+	<securecookie host="^www\.itradecimb\.co\.id$" name=".*" />
+	<securecookie host="^www\.cimbniaga\.com$" name=".*" />
+
+	<rule from="^http://cimbniaga\.com/"
+		to="https://www.cimbniaga.com/" />
```

Comment:
It would be nice if you added comments stating why you must rewrite from ^domain to www.domain in each of these rules, for the sake of webmasters reading the Atlas.